src/Commands/Remote/WPCommand.php

```diff
@@ -24,13 +24,17 @@ class WPCommand extends SSHBaseCommand
      *
      * @param string $site_env Site & environment in the format `site-name.env`
      * @param array $wp_command WP-CLI command
+     * @param array $options Commandline options
+     * @option progress Allow progress bar to be used (tty mode only)
      * @return string Command output
      *
      * @usage <site>.<env> -- <command> Runs the WP-CLI command <command> remotely on <site>'s <env> environment.
+     * @usage <site>.<env> --progress -- <command> Runs a WP-CLI command with a progress bar
```

Comment:
@TeslaDethray Is this about what you had in mind for separate @usage annotations (here and on the remote:drush command)?